frontend/src/features/resolve_differences/utils/dataEntry.ts

```diff
@@ -0,0 +1,67 @@
+import { PollingStationResults } from "@/api";
+import { FormSectionId } from "@/components/form/data_entry/state/types";
+
+type ObjectPath<T> = {
+  // For each entry, if it is an object (including optional)
+  [K in keyof T]: NonNullable<T[K]> extends object
+    ? T[K] extends unknown[]
+      ? never // Skip arrays, which are also objects
+      : `${K & string}.${Extract<keyof NonNullable<T[K]>, string>}` // "parent.child" template literal for string keys
+    : K & string; // else, the key itself when it is a string
+}[keyof T]; // Get all values together
+
+type SectionId = Extract<FormSectionId, "recounted" | "voters_votes_counts" | "differences_counts">;
+
+export type PollingStationResultsPath = NonNullable<ObjectPath<PollingStationResults>>;
+
+export interface DataEntrySection {
+  id: SectionId;
+  fields: { code?: string; path: PollingStationResultsPath }[];
+}
+
+export const sections: DataEntrySection[] = [
+  {
+    id: "recounted",
+    fields: [{ path: "recounted" }],
+  },
+  {
+    id: "voters_votes_counts",
+    fields: [
+      { code: "A", path: "voters_counts.poll_card_count" },
+      { code: "B", path: "voters_counts.proxy_certificate_count" },
+      { code: "C", path: "voters_counts.voter_card_count" },
+      { code: "D", path: "voters_counts.total_admitted_voters_count" },
+      { code: "E", path: "votes_counts.votes_candidates_count" },
+      { code: "F", path: "votes_counts.blank_votes_count" },
+      { code: "G", path: "votes_counts.invalid_votes_count" },
+      { code: "H", path: "votes_counts.total_votes_cast_count" },
+      { code: "A.2", path: "voters_recounts.poll_card_count" },
+      { code: "B.2", path: "voters_recounts.proxy_certificate_count" },
+      { code: "C.2", path: "voters_recounts.voter_card_count" },
+      { code: "D.2", path: "voters_recounts.total_admitted_voters_count" },
+    ],
+  },
```

Comment:
In the data entry form these fields are on the same page but with a header ("Toegelaten kiezers na hertelling door gemeentelijk stembureau") before A.2 to D.2:
<img width="800" alt="Screenshot 2025-04-03 at 17 06 29" src="https://github.com/user-attachments/assets/d8701b1f-4e5a-483c-b415-2e4c58e1e6ec" />

This is more clear than the current implementation of the differences table where it is not clear that fields A.2 to D.2 are recounted values:
<img width="891" alt="Screenshot 2025-04-03 at 17 06 21" src="https://github.com/user-attachments/assets/170c3ba5-8f68-428d-aaf2-cfe7790406bc" />

However, in the new models the fields A.2 to D.2 are removed. This means there won't have a need for that header and I recommend we don't implement it either.

Reply:
I thought we could try to summarize those questions to a single label that would fit in the 'Omschrijving' column. But agreed, we can design that later.

Reply:
yeah I think that would work. Especially if we add the number of the question (i.e 2.2 or 2.3) in the 'code' column
<img width="663" alt="Scherm­afbeelding 2025-04-03 om 21 17 04" src="https://github.com/user-attachments/assets/f2d00bb7-7129-411b-98c8-b922c7fdb057" />